docs/1.0. Certificate Provisioning.md

```diff
@@ -221,134 +234,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
+On connection to the target environment's network, the EST Client SHOULD attempt to discover the location of the EST Server using [DNS-SD](#dns-sd-advertisement), if EST has not been disabled or the location of the EST Server configured manually.
 
-#### Get Root CA
+#### Getting the Root CA
 
-There are two possible workflows for installing the Root CA for the target network depending on if explicit trust of EST server is **Enabled** or **Disabled**.
+There are two possible workflows for installing the Root CA for the target network depending on whether explicit trust of the EST Server is **Enabled** or **Disabled**.
 
-If explicit trust of the EST Server is **Enabled**, the EST Client SHOULD make a HTTPS request to the `/cacerts` endpoint of the EST Server for the latest Root CA of the current network. The EST Client SHOULD explicitly trust the EST Server manually configured or discovered using Unicast DNS and not perform authentication of the EST Server's TLS Certificate during the initial request to the EST Server.
+If explicit trust of the EST Server is **Enabled**, the EST Client SHOULD make an HTTPS request to the `/cacerts` endpoint of the EST Server for the latest Root CA of the current network. The EST Client SHOULD explicitly trust the EST Server manually configured or discovered using unicast DNS and not perform authentication of the EST Server's TLS Certificate during the initial request to the EST Server.
 
-If explicit trust of the EST Server is **Disabled**, the EST Client SHOULD make a HTTPS request to the `/cacerts` endpoint of the EST Server for the latest Root CA of the current network. The EST Client MUST perform authentication of the EST Server's TLS Certificate, using the list of trusted Certificate Authorities.
+If explicit trust of the EST Server is **Disabled**, the EST Client SHOULD make an HTTPS request to the `/cacerts` endpoint of the EST Server for the latest Root CA of the current network. The EST Client MUST perform authentication of the EST Server's TLS Certificate, using the EST Client's current list of trusted Certificate Authorities.
 
-If the EST Server returns a HTTP 200 response, the EST Client SHOULD add the returned Root CA to the list of trusted Certificate authorities, replacing any previously installed Root CAs which were obtained via EST. It SHOULD also store any intermediate certificates returned, which are used to form the chain of trust for certificates issued by the EST CA.
+If the EST Server returns an HTTP 200 response, the EST Client SHOULD add the returned Root CA to the list of trusted Certificate Authorities, replacing any previously installed Root CAs which were obtained via EST. It SHOULD also store any intermediate certificates returned, which are used to form the chain of trust for certificates issued by the EST CA.
```

Comment:
The phrase "[latest] Root CA [of the current network]" is clear. Does "EST CA" here mean something different?

Reply:
```suggestion
If the EST Server returns an HTTP 200 response, the EST Client SHOULD add the returned Root CA to the list of trusted Certificate Authorities, replacing any previously installed Root CAs which were obtained via EST. It SHOULD also store any intermediate certificates returned, which are used to form the chain of trust for certificates issued by the EST Server.
```

Reply:
No difference in the meaning, but think that we can just say "issued by the EST Server"

Reply:
Makes perfect sense to me. Will commit that suggestion after you've had a chance to look at the other little questions too. :-)